src/Table.js

```diff
@@ -66,24 +65,25 @@ class Table extends React.Component {
       size,
       variant,
       responsive,
-      ...props
-    } = this.props;
-
+      ...otherProps
```

Comment:
just `...props` i think... unless there’s a name collision

Reply:
It certainly could be. Mostly I was following a different example  (https://github.com/react-bootstrap/react-bootstrap/pull/3992/files) that also had otherProps, and to at least delineate that it is other props than what is being used with this stuff. 

Would require a change here, and a change on line 84.

Reply:
let’s do that. the example there already has a `props` so we needed another name